project_template/.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
\** Seething hate for git's EoL translation. **

Reply:
This should be `* -text`.

Reply:
ugh, yeah. It's way to hard to tell git to just keep things the way they are.